.travis.yml

```diff
@@ -1,6 +1,9 @@
 dist: xenial
 language: python
 python: 3.6
+branches:
```

Comment:
Why add this?

Reply:
this way, if you create a branch for httpstan and make a PR, tests are only run for the PR (now it runs duplicate amount for httpstan branches). After merge, it will run tests normally.